tests/unit/misc/test_editor.py

```diff
@@ -72,14 +72,17 @@ class TestFileHandling:
 
     """Test creation/deletion of tempfile."""
 
-    def test_ok(self, editor):
+    @pytest.mark.parametrize('remove_file', [True, False])
+    def test_ok(self, editor, remove_file, config_stub):
         """Test file handling when closing with an exit status == 0."""
+        config_stub.val.editor.remove_file = remove_file
         editor.edit("")
         filename = pathlib.Path(editor._filename)
         assert filename.exists()
         assert filename.name.startswith('qutebrowser-editor-')
         editor._proc._proc.finished.emit(0, QProcess.NormalExit)
-        assert not filename.exists()
+        if config_stub.val.editor.remove_file:
+            assert filename.exists() != config_stub.val.editor.remove_file
```

Comment:
With the change to this expression, this should now not be inside an `if ...:` anymore - if the setting is set to `False`, we want to also make sure that the file does still exist. Right now the test only check if it's gone when the setting is set to True.

Reply:
Done!